spec/support/disability_compensation_form/submissions/with_everything.json

```diff
@@ -105,6 +105,16 @@
             "activeDutyEndDate": "1990-04-05"
           }
         ],
+        "confinements": [
```

Comment:
not strictly relevant to this PR, this was a change I accidentally removed from a past PR (related to our on-going submit migration)